internal/pkg/source/source.go

```diff
@@ -109,13 +109,25 @@ func (a *Source) visitOperands(operands []*ssa.Value) {
 		if !ok || a.marked[n] {
 			continue
 		}
-		al, ok := (*o).(*ssa.Alloc)
-		if !ok || al.Comment == "slicelit" || a.config.IsSource(al.Type()) {
-			a.dfs(n)
+		al, isAlloc := (*o).(*ssa.Alloc)
```

Comment:
Nit: this line should go below the comment block.

Reply:
Done.